### DISCUSSION TITLE
Weblate running on AKS public IP

### DISCUSSION THREAD

**Discussion by binte** [WeblateOrg/weblate#5326] [Q&A] [answered]
I have just successfully deployed weblate on AKS, and everything seems to be alright. However, I am not being given a public IP, as per the below screenshot. Is there an issue, or am I just missing something?

![image](https://user-images.githubusercontent.com/5520287/106599608-974f8d80-6559-11eb-9664-372262f6e466.png)

**Comment by binte** [comment, marked as answer]
Ok, I figured this out on my own. I updated the YAML definition of the weblate service, so that it is of type LoadBalancer, and not ClusterIP. Directly on the AKS

**Reply by thadguidry** [reply]
You should mark this as answered then (even if you answered it yourself)
